src/basics.md

```diff
@@ -1297,6 +1342,9 @@ fn main() {
 [`DateTime::to_rfc2822`]: https://docs.rs/chrono/*/chrono/struct.DateTime.html#method.to_rfc2822
 [`DateTime::to_rfc3339`]: https://docs.rs/chrono/*/chrono/struct.DateTime.html#method.to_rfc3339
 [`DateTime::format`]: https://docs.rs/chrono/*/chrono/struct.DateTime.html#method.format
+[`DateTime::checked_add_signed`]: https://docs.rs/chrono/*/chrono/struct.Date.html#method.checked_add_signed
```

Comment:
Please, sort links.

> non rendered items are in sorted order (links, reference, identifiers, Cargo.toml.

from [pull request template](https://github.com/rust-lang-nursery/rust-cookbook/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

Reply:
The links in that whole section are generally unordered. That's why I followed chronological ordering grouped by topic / crate. Should I perhaps alpha-sort the whole list of API reference links, or just the ones belonging to the `chrono` crate:

Reply:
Can you alpha-sort the whole API Reference list in a separate commit in this PR?

Reply:
Yes, can do. I'll get on it as soon as I'm able.

Reply:
No problem.